tests/pact/tc.repl

```diff
@@ -3,6 +3,46 @@
 (env-data { "msgkeys": { "keys": ["k"], "pred": ">" } })
 (begin-tx)
 (define-keyset 'keyset (read-keyset "msgkeys"))
+
+(interface tctest-iface
+  (defun tcmodref-spec:bool (s:string)))
+
+(interface tctest-iface2
+  (defun tcmodref-spec2:bool (s:string)))
+
+(module tctestImpl g
```

Comment:
If you feel like it, de-camel `tctestImpl`, `tctestImpl2`, `tctestImplBoth` (ie `tctest-impl-both` ...)

Reply:
Done